hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/impl/OperationThread.java

```diff
@@ -179,22 +179,20 @@ private void process(Runnable runnable) {
     }
```

Comment:
I need to revert this. Comes from a different experiment.

Reply:
Revert?